Lib/mimetypes.py

```diff
@@ -23,10 +23,10 @@
 read_mime_types(file) -- parse one file, return a dictionary or None
 """
 
+import posixpath
 import os
 import sys
-import posixpath
```

Comment:
Any reason to move this to the top?

These are arranged in by length of the import size. 

Also, let's keep `urllib.parse` as the import instead of `urllib`.